README.md

```diff
@@ -39,6 +39,27 @@ Accepted values are: `linear`, `striped`, `raid0`, `raid1`, `raid4`, `raid5`, `r
 This is a list of volumes that belong to the current pool. It follows the
 same pattern as the `storage_volumes` variable, explained below.
 
+##### `encryption`
+This specifies whether or not the pool will be encrypted using LUKS.
+__WARNING__: Toggling encryption for a pool is a destructive operation, meaning
+             the pool itself will be removed as part of the process of
+             adding/removing the encryption layer.
+
+##### `encryption_passphrase`
+This string specifies a passphrase used to unlock/open the LUKS volume(s).
+
+##### `encryption_key_file`
+This string specifies the full path to the key file used to unlock the LUKS volume(s).
```

Comment:
path on the control node or on the managed host?

Reply:
Everything is on the managed host AFAIK. Any operations to get the key file to the managed nodes is on the user. Maybe I should note that.

Reply:
> @sergio-correia the LUKS parameters are needlessly different from the ones used in your role. https://github.com/linux-system-roles/nbde_client/pull/3/files#diff-7eeda618087b49ae876084ab6c73fdbbR9 uses `pass` and `keyfile`. Here one uses `encryption_passphrase` and `encryption_key_file`. Those names should be consistent. (I don't care which one is chosen.)

I will adopt `encryption_passphrase` and `encryption_key_file`. These seem more explicit.

Reply:
I think you an use just `passphrase` and `keyfile`. In your role it is obvious that it is encryption related (it is an encryption role). I was concerned about the gratuitous `pass` vs `passphrase` difference or `keyfile` vs. `key_file`.

Reply:
@dwlehman would it make sense to have `key_file` on the control node? `nbde_client` uses a keyfile on the control node: https://github.com/linux-system-roles/nbde_client/pull/3#discussion_r439006359 . Are the use cases different enough to justify this difference in semantics?

Reply:
I agree that it is more convenient for users and the consistency between roles is good.

Where on the managed node should we place the key files?

Reply:
> Where on the managed node should we place the key files?

@sergio-correia what do you do in nbde_client?

Reply:
I define an internal variable `__nbde_client_managed_dir` and have a task to copy the key files there and do the cleanup later. I pass this variable also to the python module, so it knows where to find the key files. It's currently defined as `/var/run/linux-system-roles.nbde_client`

Reply:
For our purposes I think we need to keep the key file around for reboots, while I (think that I) see that you are removing them in the nbde_client role after setting up nbde. For us to specify that they be on the controller necessitates developing a scheme for storing them on the managed nodes.

Reply:
Yeah, for us it's simpler that we need it to do some operations, then we don't need it anymore and can remove it.